specs_for_version.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
hmm not sure what this does, but prefer not to have it ... `rake wwtd:local` runs all the gemfiles and running a single gemfile is not that hard

Reply:
Oh! We didn't realize that this was a feature. We'll go ahead and remove the script.